workflow.go

```diff
@@ -258,13 +262,21 @@ func ExtractURLs(extractChannel <-chan *CrawlerMessageItem) (<-chan *url.URL, <-
 }
 
 func PublishURLs(ttlHashSet *ttl_hash_set.TTLHashSet, queueManager *queue.Manager, publish <-chan *url.URL) {
-	for url := range publish {
+	for uu := range publish {
 
-		u := url.String()
+		u := uu.String()
 
-		if(url.RawQuery != "") {
-			log.Debugln("Skipping URL as it has query params:", u)
-			continue
+		if(uu.RawQuery != "") {
+			values, err := url.ParseQuery(uu.RawQuery)
+			if err != nil {
+				log.Debugln("Skipping URL unable to decode query params:", u)
+				continue
+			}
+			values.Del("path")
```

Comment:
Should this be `page`, not `path`? Either way, I'd say it's worth a comment to say why we're removing this apparently magic value.

Reply:
Yes, that is correct. have updated